src/other/pyodide/PyodideLoader.tsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useState } from "react";
+
+interface PyodideLoaderProps {
+    url?: string;
+    // @ts-ignore
+    onLoad: (pyodide: any) => void;
+    triggerLoad?: boolean;
+}
+
+declare global {
+    interface Window {
+        // @ts-ignore
+        loadPyodide: () => Promise<any>;
+        // @ts-ignore
+        pyodide: any;
+    }
+}
+
+const defaultSourceUrl = "https://cdn.jsdelivr.net/pyodide/v0.24.0/full/pyodide.js";
```

Comment:
I checked the size of this file. It's really small - only 7KB. Why don't we just install it with npm?
Plus it has implementation in TS: https://github.com/pyodide/pyodide/blob/main/src/js/api.ts so we probably would be able to remove `any` everywhere in this file

Reply:
As we saw live, NPM version meant to work on Node.js, here we want to start Pyodide fully in the browser, thus the use of CDN which provides exactly that. But I'll import Pyodide from NPM for types.

Reply:
As discussed, using CDN